runtime/function.go

```diff
@@ -182,3 +185,46 @@ func initStaticMethodType(map[string]*Object) {
 	StaticMethodType.slots.Get = &getSlot{staticMethodGet}
 	StaticMethodType.slots.Init = &initSlot{staticMethodInit}
 }
+
+// classMethod represents Python 'classmethod' objects.
+type classMethod struct {
+	Object
+	callable *Object
+}
+
+func newClassMethod(callable *Object) *classMethod {
+	return &classMethod{Object{typ: ClassMethodType}, callable}
+}
+
+func toClassMethodUnsafe(o *Object) *classMethod {
+	return (*classMethod)(o.toPointer())
+}
+
+// ToObject upcasts f to an Object.
+func (m *classMethod) ToObject() *Object {
+	return &m.Object
+}
+
+func classMethodGet(f *Frame, desc, _ *Object, typ *Type) (*Object, *BaseException) {
+	m := toClassMethodUnsafe(desc)
+	if m.callable == nil {
+		return nil, f.RaiseType(RuntimeErrorType, "uninitialized classmethod object")
+	}
+	return newBuiltinFunction("_classMethodGet", func(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
```

Comment:
I think you want to return a [method object](https://github.com/google/grumpy/blob/master/runtime/method.go) rather than a function.